docs/index.rst

```diff
@@ -7,7 +7,8 @@
 Parcels documentation
 ===================================
 
-Welcome to the documentation of parcels. This page provides detailed documentation for each method, class and function. 
+Welcome to the documentation of parcels. This page provides detailed documentation for each method, class and function.
+The documentation corresponds to the latest anaconda release, for newer documentation see the docstrings in the code.
```

Comment:
[...] to the latest conda release, for [...]

Anaconda is the platform, but parcels is a conda-forge package, so writing "anaconda release' ins not entirely correct.